sys-utils/nsenter.c

```diff
@@ -119,6 +128,39 @@ static int wd_fd = -1;
 static int env_fd = -1;
 static int uid_gid_fd = -1;
 
+static int set_parent_user_ns_fd(void)
+{
+	struct namespace_file *nsfile;
+	struct namespace_file *user_nsfile;
+	int parent_ns = -1;
+
+	for (nsfile = namespace_files; nsfile->nstype; nsfile++) {
+		if (nsfile->nstype == CLONE_NEWUSER) {
+			user_nsfile = nsfile;
+		}
+
+		if (nsfile->fd == -1)
+			continue;
+
+		parent_ns = ioctl(nsfile->fd, NS_GET_USERNS);
+		if (parent_ns < 0) {
+			err(EXIT_FAILURE, _("failed to open parent ns of %s"), nsfile->name);
+			return -1;
```

Comment:
err() will abort the program, so the return after is unnecessry.

Reply:
Added